src/card/basic/card.component.js

```diff
@@ -130,6 +130,9 @@ angular.module('patternfly.card').component('pfCard', {
 
     ctrl.$onInit = function () {
       ctrl.shouldShowTitlesSeparator = (!ctrl.showTitlesSeparator || ctrl.showTitlesSeparator === 'true');
+      if (angular.isUndefined(ctrl.showSpinner)) {
+        ctrl.showSpinner = 'false';
+      }
     };
```

Comment:
Once this is a binding you can use
`ctrl.showSpinner = ctrl.showSpinner === true`

Reply:
```shouldn't I do ctrl.showSpinner = ctrl.showSpinner === false``` since we want it to be false by default?

Reply:
so you want it to be true if they pass false?

What I have above says it is true only if the specifically pass true. Any other value would resolve to false (the default).